packages/core/package.json

```diff
@@ -15,11 +15,13 @@
   },
   "files": [
     "lib",
+    "android/gradle",
     "android/src",
     "android/build.gradle",
+    "android/dependencies.gradle",
     "android/gradle.properties",
-    "ios/RNEmbrace",
```

Comment:
with this change `packages/core/ios/RNEmbrace.xcodeproj` should be removed if it's no longer needed to avoid it getting included in the package

Reply:
done